back/spec/acceptance/users_spec.rb

```diff
@@ -458,6 +458,26 @@
           json_response = json_parse(response_body)
           expect(json_response[:data].pluck(:id)).to match_array [a.id, @user.id]
         end
+
+        example 'List all blocked users' do
+          blocked_users = create_list(:user, 2, block_start_at: 30.days.ago)
+
+          settings = AppConfiguration.instance.settings
+          settings['user_blocking'] = {
+            'enabled' => true,
+            'allowed' => true,
+            'duration' => 90
+          }
+          AppConfiguration.instance.update!(settings: settings)
+
+          do_request only_blocked: true
+
+          expect(status).to eq 200
+          json_response = json_parse(response_body)
+          expect(User.count).to be > blocked_users.count # If unblocked users exist, then subsequent tests meaningful.
+          expect(json_response[:data].size).to eq 2
+          expect(json_response[:data].pluck(:id)).to match_array blocked_users.map(&:id)
+        end
```

Comment:
This is a new test (I refactored the file, so may not be obvious which tests are new)